sdc/tests/test_dataframe.py

```diff
@@ -2108,6 +2466,7 @@ def test_impl(n):
         hpat_func = self.jit(test_impl)
         pd.testing.assert_series_equal(hpat_func(n), test_impl(n))
 
+    @unittest.skip('New DataFrame structure: implement unboxing')
```

Comment:
!!!

Reply:
Thanks for this catch. Undecorated in https://github.com/IntelPython/sdc/pull/871.